lnbits/wallets/base.py

```diff
@@ -136,7 +136,7 @@ def paid_invoices_stream(self) -> AsyncGenerator[str, None]:
 
     def normalize_endpoint(self, endpoint: str, add_proto=True) -> str:
         endpoint = endpoint[:-1] if endpoint.endswith("/") else endpoint
-        if add_proto:
+        if add_proto and not endpoint.startswith("ws"):
```

Comment:
This is better. The original code does not behave correctly if `endpoint` like `wsexample.com` is provided.

```suggestion
        if add_proto:
            if endpoint.startswith("ws://") or endpoint.startswith("wss://"):
                return endpoint
```

Reply:
Sure. Thanks.